Week-1/assignment1/readme.md

```diff
@@ -0,0 +1,60 @@
+
+### F - Layout and Painting
+
+Explanation – Layout: Once the Render-Tree is constructed, then the browser starts the printing individual elements on the screen. The first browser creates the layout of each individual Render-Tree node. The layout consists of the size of each node in pixels and where (position) it will be printed on the screen. This process is called layout since the browser is calculating the layout information of each node. This process is also called reflow or browser reflow and it can also occur when you scroll, resize the window or manipulate DOM elements. Painting: after layout we have a list of geometries that need to be printed on the screen. Since elements (or a sub-tree) in the Render-Tree can overlap each other and they can have CSS properties that make them frequently change the look, position, or geometry (such as animations), the browser creates a layer for it. Creating layers helps the browser efficiently perform painting operations throughout the lifecycle of a web page such as while scrolling or resizing the browser window. Having layers also help the browser correctly draw elements in the stacking order (along the z-axis) as they were intended by the developer. Now that we have layers, we can combine them and draw them on the screen. But the browser does not draw all the layers in a single go. Each layer is drawn separately first. Inside each layer, the browser fills the individual pixels for whatever visible property the element has such as border, background color, shadow, text, etc. This process is also called as rasterization. To increase performance, the browser may use different threads to perform rasterization.
```

Comment:
Again... Please be concise. 

Use more markdown features. Please break this down into multiple paragraphs.
Very hard to understand for me what you are trying to convey. Logical flow of information is missing.
Please look into it.

Always... try to summarize a sub-topic in 4-8 sentences maximum (unless the assignment requires depth... which is not the case in my opinion...). 
That will make it easier for your peers  :-)

Reply:
Please go through the link shared by Chaitanya regarding "How Browser Works" 
They gave a good overview.